frontend/src/metabase/query_builder/selectors.js

```diff
@@ -180,8 +181,6 @@ export const getPKRowIndexMap = createSelector(
   },
 );
 
-export const getIsNew = state => state.qb.card && !state.qb.card.id;
```

Comment:
This selector was added in https://github.com/metabase/metabase/pull/2795 and was used in `frontend/src/metabase/query_builder/QueryHeader.jsx` which doesn't seem to exist anymore. I also try to find other part of the code that would relate to that file which still uses `isNew`, but couldn't find anything.